### PR TITLE
Add example.ipynb

### PR DESCRIPTION
**Pull request by y0z**
## Contributor Agreements

Please read the [contributor agreements](https://github.com/optuna/optunahub-registry/blob/main/CONTRIBUTING.md#contributor-agreements) and if you agree, please click the checkbox below.

- [x] I agree to the contributor agreements.

## Motivation

- Add example.ipynb for WhaleOptimizationAlgorithm
- Please merge https://github.com/optuna/optunahub-web/pull/32 before merging this PR

### CODE REVIEW

File: package/samplers/whale_optimization/example.ipynb
Comment (toshihikoyanase) [inline package/samplers/whale_optimization/example.ipynb:1]: Could you add ` --quiet --progress-bar off ` option to the `pip install` cell, please?

File: recipes/002_registration.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 - `__init__.py`: An initialization file. This file must implement your algorithm or import its implementation from another file, e.g., `YOUR_ALGORITHM_NAME.py`.
 - `README.md`: A description of your algorithm. This file is used to create an `web page of OptunaHub <https://hub.optuna.org/>`_. Let me explain the format of the `README.md` file later.
 - `LICENSE`: A license file. This file must contain the license of your algorithm. It should be the MIT license in the alpha version of OptunaHub.
-- `example.py`: This is optional. This file should contain a simple example of how to use your algorithm (Example: `example.py for Simulated Annealing Sampler <https://github.com/optuna/optunahub-registry/blob/main/package/samplers/simulated_annealing/example.py>`_).
+- `example.py`, `example.ipynb`: This is optional. This file should contain a simple example of how to use your algorithm (Example: `example.py for Simulated Annealing Sampler <https://github.com/optuna/optunahub-registry/blob/main/package/samplers/simulated_annealing/example.py>`_). You can provide examples in both formats.
```
Comment (toshihikoyanase) [inline recipes/002_registration.py:39]: I think this is out-of-scope of this PR, but double backticks are suitable. I'll work on it in a follow-up PR.

image

**Review by toshihikoyanase — COMMENTED** [review]
Thanks for your PR. I have some small comments.

File: package/samplers/whale_optimization/example.ipynb
Comment (y0z) [reply package/samplers/whale_optimization/example.ipynb:1]: Thank you for your feedback. Done.

**Review by toshihikoyanase — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**